stravalib/model.py

```diff
@@ -759,20 +1043,23 @@ class Activity(
     end_latlng: Optional[LatLon] = None
     map: Optional[Map] = None
     gear: Optional[Gear] = None
-    best_efforts: Optional[List[BestEffort]] = None
-    segment_efforts: Optional[List[SegmentEffort]] = None
-    splits_metric: Optional[List[Split]] = None
-    splits_standard: Optional[List[Split]] = None
+    best_efforts: Optional[List[DetailedSegmentEffort]] = None
+    segment_efforts: Optional[List[DetailedSegmentEffort]] = None
+    # Ignoring types here given there are overrides
+    splits_metric: Optional[List[Split]] = None  # type: ignore
+    splits_standard: Optional[List[Split]] = None  # type: ignore
     photos: Optional[ActivityPhotoMeta] = None
-    laps: Optional[List[ActivityLap]] = None
+    # TODO Detailed activity has this defined as - Optional[List[Lap]]
+    laps: Optional[List[Union[ActivityLap, Lap]]] = None
```

Comment:
Here since AcitivtyLap is a subclass of Lap maybe it's better to have simply Lap?

Reply:
of course. that worked - thank you!